specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -46,7 +46,7 @@ model FleetUpdate {
 }
 
 model ManagedServiceIdentityUpdate
-  is OptionalProperties<UpdateableProperties<Azure.ResourceManager.Foundations.ManagedServiceIdentity>>;
+  is OptionalProperties<UpdateableProperties<Azure.ResourceManager.Legacy.ManagedServiceIdentityV4>>;
```

Comment:
@allenjzhang I don't think we should change the Foundations to Legacy in this case, they have specified `@armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v5` before and it's a misrepresent issue between typespec and ARM definition. Customer thought they have specified v5 but we didn't generate the swagger as they thought, it seems logical to change the definition align with what they have defined before, not with the wrongful representation?